app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/demographics/NextOfKin.java

```diff
@@ -0,0 +1,49 @@
+package gov.hhs.cdc.trustedintermediary.etor.demographics;
+
+import java.util.StringJoiner;
+
+/** Contains information about a next of kin relationship. */
+public class NextOfKin {
+    private String firstName;
+    private String lastName;
+    private String phoneNumber;
+
+    public NextOfKin(final String firstName, final String lastName, final String phoneNumber) {
```

Comment:
Do we need to capture the type of next of kin in the `NextOfKin` object?

Reply:
I don't know.  No one has mentioned it, so I wasn't going to capture any additional information until we have a more concrete understanding of what's needed.